source/training/index.md

```diff
@@ -0,0 +1,62 @@
+# <i icon-name="book"></i> Training Materials
+
+```{toctree}
+:hidden:
+
+kickoff
+session1
+session2
+session3
+session4
+session5
+session6
+session7
+session8
+session9
+```
+
+The following training materials were originally developed with funding under the EU Govtech4all program.
+
+They are intended for reuse and/or adaption in future training situations.
+
+Along with the training materials are the individual graphics/assets required for extending the slides while preserving the OpenFisca brand.
+
```

Comment:
Not sure this adds much value to the end user 🙂  most likely it will stay like this for a long time and people will be happy to have access to it! Let's not diminish its value as is.

```suggestion
```